packages/utils/src/registryApi.ts

```diff
@@ -11,7 +11,7 @@ export class RegistryApi {
 
   constructor() {
     // Use window.location.origin in the browser, fallback to "/" for SSR
-    this.baseUrl = typeof window !== "undefined" ? `${window.location.origin}/api` : "/api"
+    this.baseUrl = typeof window !== "undefined" ? `${window.location.origin}/api` : "http://localhost:3000/api"
```

Comment:
This is so weird... I changed this recently, I think I had it using the NEXT_AUTH_URL, since we have that in all the envs

Reply:
Obviously localhost won't work when deployed

Reply:
Fixed